posthog/src/main/java/com/posthog/java/DefaultPostHogLogger.java

```diff
@@ -0,0 +1,37 @@
+package com.posthog.java;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class DefaultPostHogLogger implements PostHogLogger {
```

Comment:
could probably use https://docs.oracle.com/javase/8/docs/api/java/util/logging/Logger.html but having PostHog logger is also fine

Reply:
yep, I'm far enough away from java ecosystem that I thought if we have an interface then folk can use just about whatever they want and avoid coupling ourselves to something only for people to say "but why not OmniLogger.io"